frontend/sw360-portlet/src/main/webapp/html/admin/fossyAdmin/view.jsp

```diff
@@ -77,6 +91,11 @@
             url: url,
             success: function (data) {
                 $resultElement.text(data.result);
+                if(data.result == 'SUCCESS'){
```

Comment:
I think we might have a problem here: Since you change the fossology state in this generic ajax method, the state will be also be changed (only in client view here, not on server) when a script has been deployed successfully, which is the other operation on this page that uses this ajax method.

Reply:
Thanks for pointing it out. Have corrected this one.